test/view_component/integration_test.rb

```diff
@@ -359,6 +359,11 @@ class IntegrationTest < ActionDispatch::IntegrationTest
     assert_select(".item.normal", count: 2)
 
     assert_select(".footer.text-blue h3", text: "This is the footer")
+
+    title_node = Nokogiri::HTML.fragment(response.body).css(".title").to_html
+    expected_title_html = "<div class=\"title\">\n    <p>This is my title!</p>\n  </div>"
```

Comment:
What would we have to do to get this to succeed?

```suggestion
    expected_title_html = "<div class=\"title\"><p>This is my title!</p></div>"
```

Reply:
@jonrohan we'd have to strip the rendered output for the component itself. It's certainly doable 😄

Reply:
The browser wouldn't treat these differently, but they are 10 bytes in size difference.

Reply:
I recently made some changes to `asset_dom_equal` so that it ignores white space between elements. I’ve been using it to test the output of my view components and it has been working well. It’s in master but not a released version of the gem, but might be something to look at for this https://github.com/rails/rails-dom-testing/pull/84